.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- 👋 Hi, thanks for sending a PR to LearningTypescript! 💖.
+Please fill out all fields below and make sure each item is true and [x] checked.
+Otherwise we may not be able to review your PR. -->
```

Comment:
https://github.com/LearningTypeScript/projects/pull/13#issuecomment-1133926807:

> Hey @JoshuaKGoldberg , I copied the exact same one at typescript-eslint you recomended. Also I saw the greeting was commented out.. I left it commented out. Is there any reason for that ?

Yup, that's intentional. We don't need that greeting to show up in text on every PR. But people should see it when they fill out the PR description.

Reply:
Oh I see. That's cool